src/test/java/it/unisa/Amigo/gruppo/controller/GruppoControllerTest.java

```diff
@@ -53,9 +52,9 @@ void visualizzaMembriSupergruppo() throws Exception {
         List<Persona> expectedPersone = new ArrayList<>();
         expectedPersone.add(expectedPersona);
 
-        when(gruppoService.visualizzaPersonaLoggata()).thenReturn(expectedPersona);
+        when(gruppoService.getAuthenticatedUser()).thenReturn(expectedPersona);
         when(gruppoService.findSupergruppo(expectedSupergruppo.getId())).thenReturn(expectedSupergruppo);
```

Comment:
Una domanda, a che serve l'attributo "type" in Supergruppo? A distinguere se è Gruppo o Commissione?